### DISCUSSION TITLE
A research for generating PR checklists in Pull Request Template

### DISCUSSION THREAD

**Discussion by baishuotong** [jaredpalmer/formik#3920] [Show and tell]
Dear maintainers:
Our team is currently developing a tool for automatically generating PR checklists in Pull Request Template. The tool generates a set of PR rules based on the contribution guidelines in your project, which external contributors may be expected to adhere to. Below are the results of our generation:

**Test**: Unit yarn Testing.
**Documentation**: Suggest modifications to documentation.
**Style**: Share workflows or insights through guides.
**Branch**: Branch with upstream.

Also, we suggest you add some checklist notes about **title**, **description**, **commit message**, **contributing**, and **cla**, which are also important for submitting PRs according to our analysis.

We hope you and your team can provide feedback on our generated output.
Best regards.

**Comment by lsm2842035890** [comment]
Good idea，we will consider it!
